build_deps/toolchains/rocksdb/rocksdb.BUILD

```diff
@@ -0,0 +1,24 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "make")
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+licenses(["notice"])  # BSD
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+make(
+    name = "rocksdb",
+    args = [
+        "EXTRA_CXXFLAGS=\"-fPIC -D_GLIBCXX_USE_CXX11_ABI=0\"",
```

Comment:
Not all tensorflow are compiled with “-D_GLIBCXX_USE_CXX11_ABI=0”. Replace it by:
```
load(
    "@local_config_tf//:build_defs.bzl",
    "D_GLIBCXX_USE_CXX11_ABI",
)
```